packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.constant.js

```diff
@@ -0,0 +1,6 @@
+export const PROMO_DISPLAY = {
+  PROMO_FTTH_2020: 'FTTH_MIGRATION_AND_ADRESSMOVE_2020',
```

Comment:
Shouldn't be:
```suggestion
  PROMO_FTTH_2020: 'FTTH_MIGRATION_AND_ADDRESSMOVE_2020',
```
?

Reply:
no it's the real name of the promotion return from API